src/codegen/llvm.zig

```diff
@@ -428,7 +428,18 @@ pub const Object = struct {
         if (!options.strip) {
             switch (options.target.ofmt) {
                 .coff => llvm_module.addModuleCodeViewFlag(),
-                else => llvm_module.addModuleDebugInfoFlag(),
+                else => {
+                    const dwarf_format: std.builtin.DwarfFormat = switch (options.target.isDarwin()) {
+                        // For Darwin targets, using DWARF64 is problematic, so stick to DWARF32
+                        true => .dwarf32,
```

Comment:
To be honest, we don't have to special case Darwin as by default we use dwarf32, and I would assume the user would know the requirements of each platform if they override it using the flag so let's let them.

Reply:
Done :)